data/json/overmap/overmap_mutable/stream.json

```
@@ -0,0 +1,47 @@
+[
+  {
+    "type": "overmap_special",
+    "id": "stream_mutable",
+    "subtype": "mutable",
+    "locations": [ "field", "forest", "swamp" ],
+    "occurrences": [ 1, 1 ],
```

Zero need to be a mandatory special, also this will allow more than one to spawn.
```suggestion
    "occurrences": [ 0, 2 ],
```